src/main/java/eu/deltasource/internship/hotel/service/GuestService.java

```diff
@@ -92,14 +89,34 @@ public void createGuest(int guestId, String firstName, String lastName, Gender g
 	 * Asserts whether the data inside the specified guest is valid.
 	 * If the object has null value or the id is negativa
 	 *
-	 * @param item Represents the guest that is to be asserted.
-	 * @return Returns a boolean answer based on the assertion's outcome.
+	 * @param items Represents the guests that are to be asserted.
+	 */
+	private void assertGuest(Guest... items) {
+		for (Guest guest : items) {
+			if (guest == null || guest.getGuestId() < 0) {
+				throw new ItemNotFoundException("guestItem is invalid !");
+			}
+			if (guest.getFirstName().contains(" ") || guest.getLastName().contains(" ")) {
```

Comment:
not really needed, maybe empty string would be enough maybe just .trim().isEmpty() ?